src/toxcore/messenger/packet/file_control.rs

```diff
@@ -0,0 +1,148 @@
+/*! FileControl struct.
+It is used to control transferring file to a friend.
+*/
+
+use nom::{le_u8, be_u64};
+
+use crate::toxcore::binary_io::*;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[repr(u8)]
+/// Whether this node is sender or receiver of file data packet
```

Comment:
What does `this node` mean?

Reply:
/// Whether I am a sender or receiver of file data packet